src/core/thread/key_manager.hpp

```diff
@@ -77,8 +77,17 @@ class SecurityPolicy : public otSecurityPolicy, public Equatable<SecurityPolicy>
      */
     static constexpr uint8_t kVersionThresholdOffsetVersion = 3;
 
-    static constexpr uint16_t kMinKeyRotationTime     = 1;   ///< The minimum Key Rotation Time in hours.
-    static constexpr uint16_t kDefaultKeyRotationTime = 672; ///< Default Key Rotation Time (in unit of hours).
+    /**
+     * Default Key Rotation Time (in unit of hours).
+     *
+     */
+    static constexpr uint16_t kDefaultKeyRotationTime = 672;
+
+    /**
+     * Minimum Key Rotation Time (in unit of hours).
+     *
+     */
+    static constexpr uint16_t kMinRotationTime = 2;
```

Comment:
```suggestion
    static constexpr uint16_t kMinKeyRotationTime = 2;
```

adding "Key" here to keep aligned with other names?

Reply:
Changed in new push.